scripts/git-hooks/pre-commit-minimum

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+##
+# Automatically run tests when commiting and prevent committing if tests fail.
+# Intended for installation into local .git/hooks folder so it runs on the
+# developers machine. Tests are run locally and expect the developer's machine
+# to have PHP and PHPCS installed. Designed to run very quickly.
+#
+# $ cd .git/hooks; ln -s ../../scripts/git-hooks/pre-commit-minimum pre-commit
+##
+
+# Really fast check if syntax is at all parseable
+for FILE in $(git diff --cached --name-only); do
+if [[ "$FILE" =~ \.php$ ]]; then
+    php -l "$FILE" 1> /dev/null
+    if [[ $? -ne 0 ]]; then
+        echo -e "\e[1;33mAborting commit: PHP code contains syntax errors\e[0m" >&2
```

Comment:
These errors are formatted funnily without colors on my console:
```
\e[1;33mAborting commit: PHP code contains syntax errors\e[0m
```
Also, the PHP code violates coding standards error is also displayed in all cases that `php -l` returns a nonzero exist status, e.g. when the file does not exist, but is that a common scenario?

Reply:
This is as it was in the old version, no new code here. Since I don't know an easy fix for this, I will put this in now and maybe later return to iterate on it.